kubernetes/bosh-cloud-config.yml

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
sure, i think we want to optimize for 95% case. rest of cases could be done by having ppl write their own custom ops file.